js/freewheel.js

```diff
@@ -464,28 +461,6 @@ OO.Ads.manager((function(_, $) {
     }, this);
 
     /**
-     * Called by the ad manager controller.  Pauses the current ad.
-     * @public
-     * @method Freewheel#pauseAd
-     * @param {object} ad The ad to pause
-     */
-    this.pauseAd = function(ad) {
-      OO.log("FW: Pausing the ad");
-      amc.ui.adVideoElement[0].pause();
-    };
-
-    /**
-     * Called by the ad manager controller.  Resumes the current ad.
-     * @public
-     * @method Freewheel#resumeAd
-     * @param {object} ad The ad to resume
-     */
-    this.resumeAd = function(ad) {
-      OO.log("FW: Resuming the ad");
-      amc.ui.adVideoElement[0].play();
-    };
-
-    /**
```

Comment:
I'm actually leaning more towards having them both in. Since these will be public, someone might notice why they are missing and won't have an immediate answer.  Or we could not include them but have a comment (in code or documentation) saying why pause and resume are not implemented by specific ad managers.